### ISSUE TITLE
Issue #424 is back

### ISSUE THREAD

**Issue by vsubramani** [google/google-api-php-client#792]
The issue mentioned here about undefined curl constants is back.

 https://github.com/google/google-api-php-client/issues/424

This might have been reintroduced from the following refactoring change by bshaffer:

https://github.com/google/google-api-php-client/commit/48dfd9571854d35b2fda1db15b918e6976d7d1e6

**Comment by bshaffer** [comment]
Thank you for reporting this. Please include the error and stack trace.

**Comment by jfradj** [comment]
I confirm, the issue is back.

In v2.0.0-RC4, the Google_Task_Runner class declares (lines [79, 83]):
```PHP
CURLE_COULDNT_RESOLVE_HOST => self::TASK_RETRY_ALWAYS,
CURLE_COULDNT_CONNECT => self::TASK_RETRY_ALWAYS,
CURLE_OPERATION_TIMEOUTED => self::TASK_RETRY_ALWAYS,
CURLE_SSL_CONNECT_ERROR => self::TASK_RETRY_ALWAYS,
CURLE_GOT_NOTHING => self::TASK_RETRY_ALWAYS
```

Theses constants are available **only** if the curl extension is loaded.

For insta

**Comment by bshaffer** [comment]
Fixed in #796
